demisto_sdk/commands/pre_commit/pre_commit_command.py

```diff
@@ -89,25 +89,49 @@ def _get_hooks(pre_commit_config: dict) -> dict:
         hooks = {}
         for repo in pre_commit_config["repos"]:
             for hook in repo["hooks"]:
-                hooks[hook["id"]] = hook
+                hooks[hook["id"]] = {"repo": repo, "hook": hook}
                 # if the hook has a skip key, we add it to the SKIPPED_HOOKS set
                 if hook.pop("skip", None):
                     SKIPPED_HOOKS.add(hook["id"])
         return hooks
 
+    def handle_python2_files(self, unit_test: bool) -> List[str]:
+        """
+        This function handles the python2 files.
+        Files with python2 run only the "run-unit-tests" hook.
+        """
+        python2_files = self.python_version_to_files.pop(DEFAULT_PYTHON2_VERSION, None)
+        if not python2_files:
+            return []
+        if not unit_test:
+            logger.info(
+                f"Skipping pre-commit with Python {DEFAULT_PYTHON2_VERSION} because unit-tests were not selected"
+            )
+            return []
+        python2_files_string = [str(file) for file in python2_files]
+        for hook in self.hooks.values():
+            if hook["hook"]["id"] == "run-unit-tests":
```

Comment:
```suggestion
            if hook["hook"]["id"] in {"run-unit-tests", "mypy"}:
```

We can run mypy on python2 files, for now.